echo-notifications/src/main/java/com/netflix/spinnaker/echo/notification/GoogleCloudBuildNotificationAgent.java

```diff
@@ -43,23 +44,29 @@ public class GoogleCloudBuildNotificationAgent implements EventListener {
 
   @Override
   public void processEvent(Event event) {
+    log.info("Process Event: {}", event.getDetails());
     if (event.getDetails() != null && event.getDetails().getType().equals("googleCloudBuild")) {
       MessageDescription messageDescription =
           (MessageDescription) event.getContent().get("messageDescription");
-      retrySupport.retry(
-          () ->
-              AuthenticatedRequest.allowAnonymous(
-                  () ->
-                      igorService.updateBuildStatus(
-                          messageDescription.getSubscriptionName(),
-                          messageDescription.getMessageAttributes().get("buildId"),
-                          messageDescription.getMessageAttributes().get("status"),
-                          RequestBody.create(
-                              messageDescription.getMessagePayload(),
-                              MediaType.parse("application/json")))),
-          5,
-          2000,
-          false);
+      try {
+        retrySupport.retry(
+            () ->
+                AuthenticatedRequest.allowAnonymous(
+                    () ->
+                        Retrofit2SyncCall.execute(
+                            igorService.updateBuildStatus(
+                                messageDescription.getSubscriptionName(),
+                                messageDescription.getMessageAttributes().get("buildId"),
+                                messageDescription.getMessageAttributes().get("status"),
+                                RequestBody.create(
+                                    messageDescription.getMessagePayload(),
+                                    MediaType.parse("application/json"))))),
+            5,
+            2000,
+            false);
+      } catch (Exception e) {
+        log.error("Failed to update google build status: {}", e);
```

Comment:
Hrmm, wouldn't this NORMALLY log elsewhere on the processing?   E.g. https://github.com/spinnaker/echo/blob/4c0e2959f1a9613d23011e316df98b5862bf44e5/echo-core/src/main/java/com/netflix/spinnaker/echo/events/EventPropagator.java#L80 should have caught it there... if it wasn't handled someplace else on the other event listeners.  Kinda makes me wonder if there's some other missing stuff in echo.